tests/lib/rules/no-restricted-imports.js

```diff
@@ -211,7 +269,13 @@ ruleTester.run("no-restricted-imports", rule, {
                 message: "Don't import 'foo'."
             }]
         }],
-        errors: [{ message: "'fs' import is restricted from being used. Don't import 'foo'.", type: "ExportNamedDeclaration" }]
+        errors: [{
+            message: "'foo' import from 'fs' is restricted. Don't import 'foo'.",
+            type: "ExportNamedDeclaration",
+            line: 1,
+            column: 9,
+            endColumn: 17
```

Comment:
Could you please clarify the `endColumn` property. Should it be used from 'root' specifier or from imported/local?

Reply:
`column` and `endColumn` are just 1-based columns in the source code, unaware of the context. Also, `endColumn` has +1 added.

Or maybe I didn't understand the question. Did you mean should the error highlight `foo as b` or just `foo` in this particular example? Both choices make sense, I think it's ok to highlight `foo as b` as it is implemented.

Reply:
> Did you mean should the error highlight foo as b or just foo in this particular example?

That was my question exactly 👍

Reply:
This is how it looks in VSCode:

![image](https://user-images.githubusercontent.com/44349756/71936359-7deadf00-31a9-11ea-88c3-029fb1cacf8b.png)

I think it's okay. It would have made sense to report just `foo` as well, but then we should probably report just `*` in `* as foo`, and that would add some extra code to find the token, so I believe it isn't necessary.